server/src/domain/facial-recognition/facial-recognition.services.ts

```diff
@@ -27,7 +27,7 @@ export class FacialRecognitionService {
     @Inject(ISearchRepository) private searchRepository: ISearchRepository,
     @Inject(IStorageRepository) private storageRepository: IStorageRepository,
   ) {
-    this.configCore = new SystemConfigCore(configRepository);
+    this.configCore = new SystemConfigCore(systemConfig);
```

Comment:
SystemConfigCore is instantiated 9 times and 7 of the 9 use `configRepository`. I think we should standardize on that name. What do you think?

Reply:
Hmm, you've got a point there. I didn't notice the different variable names. I'll rename it to `configRepository`.